### ISSUE TITLE
Scoreboard issue.

### ISSUE THREAD

**Issue by Satton2** [TTT-2/TTT2#239]
Well, this was an average round but one player was able to see (fake) traitor in the scoreboard while other players didn't. He killed him and he was an innocent. He sent me a screenshot of his scoreboard and the (fake) traitor was really marked as traitor while on my scoreboard he wasn't.
My screenshot:
![20190127000012_1](https://user-images.githubusercontent.com/23385240/51792971-8598ba00-21ca-11e9-8498-be0fb19a293e.jpg)
His screenshot:
![4000_screenshots_20190127000013_1](https://user-images.githubusercontent.com/23385240/51792975-8fbab880-21ca-11e9-97eb-a3e3371ad53c.jpg)
There wasn't any errors (at least from client-side).

**Comment by dok441** [comment]
![0](https://user-images.githubusercontent.com/7480655/52015466-ddedf580-2503-11e9-8b62-9a28ac15e78f.png)
Так же хочу подтвердить это, как мы видим, предатель розовый, хотя он должен быть красным.

**Comment by Satton2** [comment]
![20190201205305_1](https://user-images.githubusercontent.com/23385240/52150396-e881c980-2680-11e9-9bfc-4e9825de4c48.jpg)
![20190201205310_1](https://user-images.githubusercontent.com/23385240/52150398-e91a6000-2680-11e9-89e2-43d58ca53913.jpg)
Well, this happens again. I have connected to the server and become a Vampire. Then I saw a Jackal ring on one of the players (I don't remember is he was a real Jackal or not) and a Survivalist in the scoreboard.
I guess this happens when you connect to the server and spawns when you still joining to the server. Because of that, player gets a fake information from server (maybe from the previous round) and doesn't get a real one. Of course this is just my theory.
